Sources/SwiftCompilerPluginMessageHandling/Diagnostics.swift

```diff
@@ -109,8 +109,17 @@ extension PluginMessage.Diagnostic {
               to: .afterTrailingTrivia
             )
             text = newTrivia.description
+          case .ensuringTrailingSpace(let token):
+            if !token.trailingTrivia.contains(where: { $0.isSpaceOrTab }) {
+              range = sourceManager.range(
+                of: Syntax(token),
+                from: .beforeTrailingTrivia,
+                to: .afterTrailingTrivia
+              )
+              text = token.trailingTrivia.appending(.space).description
+            }
```

Comment:
Is there a test package for `SwiftCompilerPluginMessageHandling`? 
Don't look like there is one.

Reply:
This used to be part of `SwiftCompilerPlugin` and I think it’s still tested as part of `SwiftCompilerPluginTests`. If you want to write tests for it, feel free to add a test target.